src/app/ClusterStateCache.cpp

```diff
@@ -66,14 +67,58 @@ CHIP_ERROR AttributeCache::UpdateCache(const ConcreteDataAttributePath & aPath,
     return CHIP_NO_ERROR;
 }
 
-void AttributeCache::OnReportBegin()
+CHIP_ERROR ClusterStateCache::UpdateEventCache(const EventHeader & aEventHeader, TLV::TLVReader * apData, const StatusIB * apStatus)
+{
+    if (apData)
+    {
+        //
```

Comment:
What happens if we already have an `mEventStatusCache` entry for this path?  Should that entry be removed here?

Reply:
Resolved offline.